calibrate.py

```diff
@@ -87,20 +153,39 @@ def parse_args():
-                        help="Sets the maximum epiploar allowed with rectification")
+    # parser.add_argument("-ep", "--maxEpiploarError", default="1.0", type=float, required=False,
+    #                     help="Sets the maximum epiploar allowed with rectification")
     parser.add_argument("-cm", "--cameraMode", default="perspective", type=str,
                         required=False, help="Choose between perspective and Fisheye")
     parser.add_argument("-rlp", "--rgbLensPosition", default=135, type=int,
                         required=False, help="Set the manual lens position of the camera for calibration")
-    parser.add_argument("-fps", "--fps", default=10, type=int,
-                        required=False, help="Set capture FPS for all cameras. Default: %(default)s")
     parser.add_argument("-cd", "--captureDelay", default=5, type=int,
                         required=False, help="Choose how much delay to add between pressing the key and capturing the image. Default: %(default)s")
     parser.add_argument("-d", "--debug", default=False, action="store_true", help="Enable debug logs.")
     parser.add_argument("-fac", "--factoryCalibration", default=False, action="store_true",
                         help="Enable writing to Factory Calibration.")
-    
+    parser.add_argument("-osf", "--outputScaleFactor", type=float, default=0.5,
+                        help="set the scaling factor for output visualization. Default: 0.5.")
+    parser.add_argument('-fps', '--framerate', type=float, default=10,
+                        help="FPS to set for all cameras. Default: %(default)s")
+    parser.add_argument('-ab', '--antibanding', default='50', choices={'off', '50', '60'},
+                        help="Set antibanding/antiflicker algo for lights that flicker at mains frequency. Default: %(default)s [Hz]")
+    parser.add_argument('-scp', '--saveCalibPath', type=str, default="",
+                        help="Save calibration file to this path")
+    parser.add_argument('-dst', '--datasetPath', type=str, default="dataset",
+                        help="Path to dataset used for processing images")
+    parser.add_argument('-mdmp', '--minDetectedMarkersPercent', type=float, default=0.5,
+                        help="Minimum percentage of detected markers to consider a frame valid")
+    parser.add_argument('-nm', '--numMarkers', type=int, default=None, help="Number of markers in the board")
+    parser.add_argument('-mt', '--mouseTrigger', default=False, action="store_true",
+                        help="Enable mouse trigger for image capture")
+    parser.add_argument('-nic', '--noInitCalibration', default=False, action="store_true",
+                        help="Don't take the board calibration for initialization but start with an empty one")
+    parser.add_argument('-dismin', '--displayMinimum', default=False, action="store_true",
+                        help="Display the new minimum every second while taking pictures")
```

Comment:
what minimum is this @moratom

Reply:
I used argument for display of messages. Now it is changed to enableDebugMessageSync.

Reply:
make it something like `debugMinoffsetSync` add better help explaination